.github/workflows/release.yml

```diff
@@ -1,50 +1,111 @@
-name: Release
+name: Release From Branch
 
 env:
   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
 on:
   workflow_dispatch:
     inputs:
-      increment:
-        description: "SemVer Increment"
-        required: true
-        default: "prerelease"
-        type: choice
-        options:
-          - prerelease
-          - prepatch
-          - preminor
-          - premajor
-          - patch
-          - minor
-          - major
+      release_branch:
+        description: "Staging branch to run release from"
+
 jobs:
-  build:
-    runs-on: macos-latest
+  smoke-test:
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
         with:
-          token: ${{ secrets.TURBOBOT }}
+          ref: ${{ inputs.release_branch }}
       - uses: ./.github/actions/setup-node
         with:
           enable-corepack: false
       - uses: ./.github/actions/setup-go
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Test
+        run: pnpm -- turbo run test --filter=cli --color
 
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
+  darwin:
```

Comment:
How about `publish-on-darwin` or something? Just `darwin` sounds like it means something to Github Actions.

Reply:
I added a name to the job instead, in the hopes that that will be clearer than something hyphenated, and it will show up nicely in the UI